test/call-attribute.ll

```diff
@@ -0,0 +1,17 @@
+; REQUIRES: pass-plugin
```

Comment:
I missed that for the previously merged PR, my bad. But it's a part of a separate commit. Please preserve the same git history as on main, aka do not merge several patches into a single commit and please preserve the original commit messages and names